lib/cronitor/monitor.rb

```diff
@@ -187,4 +180,12 @@ def clean_params(params)
   def self.symbolize_keys(obj)
     obj.inject({}){|memo,(k,v)| memo[k.to_sym] = v; memo}
   end
+
+  def self._headers
+    {
+     'Content-Type': 'application/json',
+     'User-Agent': 'cronitor-ruby',
+     'Cronitor-Version': Cronitor.api_version
+    }
+  end
```

Comment:
Hmmm, these are defined twice? Once as part of `Cronitor#config` in `lib/cronitor/config.rb`, and again here.

Seems like it should be one or the other, but I’m not sure which at the moment.

Reply:
ah good catch!